stdlib/public/Platform/Platform.swift

```diff
@@ -103,8 +103,7 @@ public var errno : Int32 {
   get {
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS) || os(FreeBSD)
     return __error().pointee
-//FIXME: os(Windows) should be replaced, such as triple(Cygwin)
```

Comment:
`os(Windows)` will not be a rare occurrence. You can use the Windows API from both MSVC and Cygwin environments, and this will be incredibly common. If we're talking about the POSIX API, then `env(Cygnus)` would be more similar to Linux/FreeBSD, etc., but if we're talking about the Win32 API, then `env(Cygnus)` would be completely dissimilar to Linux/FreeBSD, etc., and more similar to the `env(MSVC)` case.